packages/webview_flutter/webview_flutter_android/pigeons/android_webview.dart

```diff
@@ -141,6 +141,8 @@ abstract class WebViewHostApi {
 
   int getScrollY(int instanceId);
 
+  List<int> getScrollPosition(int instanceId);
```

Comment:
Let's make a `WebViewPoint` class in the Pigeon definition, and return that, so it's self-documenting what the values mean, and we don't have to worry about things like a length mismatch. (It's less efficient over the wire for now, but it's not a big deal, and it's also something we plan to fix in future versions of Pigeon anyway.)

We could consider using it for `scrollTo` as well.